ReactWindows/ReactNative.Shared/Modules/Core/ReactChoreographer.cs

```diff
@@ -79,12 +82,24 @@ public static ReactChoreographer Instance
             }
         }
 
-        private static bool HasCoreWindow
+#if WINDOWS_UWP
+        /// <summary>
+        /// Factory for choreographer instances associated with non main-view dispatchers.
+        /// </summary>
+        public static ReactChoreographer CreateSecondaryInstance(CoreApplicationView view)
+        {
+            var instance = new ReactChoreographer();
+            instance._applicationView = view;
```

Comment:
Can we add another private constructor that takes a CoreApplicationView instead?

Reply:
That way `_applicationView` can be readonly.

---
In reply to: [165943070](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r165943070) [](ancestors = 165943070)